src/main/drivers/sdcard.c

```diff
@@ -572,11 +573,20 @@ void sdcard_init(const sdcardConfig_t *config)
 
     sdcard.useDMAForTx = config->useDma;
     if (sdcard.useDMAForTx) {
+
+        const dmaChannelSpec_t *chanspec = dmaGetChannelSpec(DMA_PERIPH_SPI_TX, config->device + 1, SDCARD_DMA_CHANOPT);
```

Comment:
Probably want to use `config->dmaopt` as the last parameter here.

Reply:
Yeap!